jax/core.py

```diff
@@ -664,10 +699,6 @@ def pp_eqn_compact(primitive_name, params):
 def pp_eqn(eqn):
   lhs = pp_vars(eqn.outvars)
   pp_subexpr = pp('')
-  if eqn.bound_subjaxpr:
```

Comment:
Not necessary for this PR, but it'd be nice to pretty-print jaxprs in params better, at least for call primitives. Right now they get indented much more than subjaxprs were (in this block being deleted).

Reply:
I plan to revisit the printing of Jaxprs separately.